catalog/spatial/registry/registry-federation-admin-service-impl/src/test/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntriesTest.java

```diff
@@ -263,14 +278,38 @@ public void testNoUpdatesOnOlder() throws Exception {
 
         refreshRegistryEntries.setRegistryStores(Collections.singletonList(registryStore));
         when(registryStore.isPullAllowed()).thenReturn(true);
-        when(registryStore.getId()).thenReturn(TEST_ID);
         when(registryStore.isAvailable()).thenReturn(true);
 
         refreshRegistryEntries.refreshRegistryEntries();
 
         verify(federationAdminService, never()).updateRegistryEntry(any(Metacard.class));
     }
 
+    @Test
+    public void testDestroy() throws Exception {
+        when(executorService.awaitTermination(anyLong(), any(TimeUnit.class))).thenReturn(true);
+        refreshRegistryEntries.destroy();
+        verify(executorService, times(1)).awaitTermination(anyLong(), any(TimeUnit.class));
```

Comment:
We should verify that the termination timeout is set to 60 seconds in at least one test.